cluster-scope/overlays/prod/moc/smaug/objectbucketclaims/opf-datacatalog-noobaa.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: objectbucket.io/v1alpha1
+kind: ObjectBucketClaim
```

Comment:
can you add the obc's to the smaug kustomization? (maybe add a `kustomization.yaml` to `objectbucketclaims` folder first).